setup.cfg

```diff
@@ -51,35 +51,40 @@ keywords =
 
 [options]
 zip_safe = True
-install_requires = 
+install_requires =
+    appdirs ~= 1.4
 	depinfo
+    diskcache ~= 5.0
 	future
+    httpx ~= 0.14
```

Comment:
Would it be much harder to download using the native Python http module? This an unstable version so it would probably break quite often.

Reply:
AFAIK, the httpx API is fairly stable by now and will soon see a 1.0 release. You're right that it's probably not too much harder, although it is quite convenient.